quickstart/cpp/windows/text-to-speech/README.md

```diff
@@ -36,5 +39,5 @@ To debug the app and then run it, press F5 or use **Debug** \> **Start Debugging
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-text-to-speech-cpp-windows)
+* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-text-to-speech?tabs=terminal&pivots=programming-language-cpp)
```

Comment:
Please remove the `en-us/` part from all the links; the docs site will automatically add in the appropriate region where appropriate.